metaflow/plugins/azure/includefile_support.py

```diff
@@ -57,13 +57,17 @@ def get(self, key=None, return_missing=False):
         with storage.load_bytes([short_key]) as load_result:
```

Comment:
As a quick note from the previous lines: why do you only support return_missing=True?

Reply:
That is a good question - and it took me a while to remember.  I added a comment inline for posterity.